src/SIL.XForge.Scripture/ClientApp/src/assets/i18n/non_checking_en.json

```diff
@@ -146,6 +146,16 @@
     "transcelerator_some_questions_already_imported": "Note: Some selected questions have already been imported. Questions that have been edited in Transcelerator will be updated in Scripture Forge, and exact duplicates will be skipped.",
     "update_transcelerator": "The version of Transcelerator used in this project is not supported. Please update to at least Transcelerator version 1.5.3."
   },
+  "join": {
+    "error_occurred_login": "An error occurred trying to join the project",
+    "invited_to_join": "You've been invited to join {{ boldStart }}{{ projectName }}{{ boldEnd }} project as a {{ boldStart }}{{ role }}{{ boldEnd }}",
```

Comment:
The more I think about this the more I think it's not important to state the role. While it might be helpful to tell the user specifically what they're expected to do (e.g. comment or answer questions), this is probably not the place for that. It would be either in the invitation, or after logging in. Removing it would also solve the localization issue.